Python/ceval.c

```diff
@@ -3650,6 +3649,76 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             NOTRACE_DISPATCH();
         }
 
+        TARGET(LOAD_ATTR_CLASS_MUTABLE_DESCRIPTOR) {
```

Comment:
Is this common enough to justify its own specialization, especially as we need to do much of the work of general lookup?

Reply:
According to the stats on the faster-cpython repo, this represents 5.7% of the remaining failures. In contrast, properties are 13.8%. Class method objects are quite close, at 6.2%. 

Here's a microbenchmark, with no PGO nor LTO, but it's a Windows release build (similar to -O3).
```
python -m timeit -s "
from enum import Enum

class Colours(Enum):
 RED = 1
" "Colours.RED"

# on MAIN:
2000000 loops, best of 5: 121 nsec per loop
2000000 loops, best of 5: 120 nsec per loop
2000000 loops, best of 5: 120 nsec per loop

# on this PR:
5000000 loops, best of 5: 77.2 nsec per loop
5000000 loops, best of 5: 73.5 nsec per loop
5000000 loops, best of 5: 72.1 nsec per loop
```
With PGO and LTO, I expect the difference to be less dramatic. But I still expect this form to be ~20% faster than the base version.

Reply:
Do you have a non-enum example?

enums members should be simple class attributes.
https://github.com/python/cpython/issues/93910

Reply:
I don't. This opcode was specifically design with enum in mind.